src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs

```diff
@@ -760,7 +772,7 @@ public async Task UpdateAPIRevisionAsync(APIRevisionListItemModel revision)
         public async Task AutoArchiveAPIRevisions(int archiveAfterMonths)
         {
             var lastUpdatedDate = DateTime.Now.Subtract(TimeSpan.FromDays(archiveAfterMonths * 30));
-            var manualRevisions = await _apiRevisionsRepository.GetAPIRevisionsAsync(lastUpdatedOn: lastUpdatedDate, apiRevisionType:  APIRevisionType.Manual);
+            var manualRevisions = await _apiRevisionsRepository.GetAPIRevisionsAsync(lastUpdatedOn: lastUpdatedDate, apiRevisionType:  APIRevisionType.PullRequest);
```

Comment:
what this indented all along and you discovered a bug? or is there other reason to change it?
Also, consider changing the name to `pullRequestRevisions`

Reply:
It's not required. I am reverting it.